src/main/webapp/app/course/manage/course-update.component.html

```diff
@@ -1,19 +1,38 @@
 <div class="row justify-content-center">
     <div class="col-md-8">
         <form [formGroup]="courseForm" (ngSubmit)="save()" #formDir="ngForm">
-            <div class="headline">
+            <div class="container headline">
+                <input type="file" #fileInput style="display: none" (change)="setCourseImage($event)" />
                 @if (croppedImage) {
-                    <jhi-secured-image [src]="croppedImage" [cachingStrategy]="CachingStrategy.LOCAL_STORAGE" />
+                    <div class="image-with-icons">
+                        <jhi-secured-image class="secured-image" [src]="croppedImage" [cachingStrategy]="CachingStrategy.LOCAL_STORAGE" />
+
+                        <button type="submit" class="btn icon-button-wrapper icon-button-1" (click)="deleteCourseIcon()">
+                            <fa-icon [icon]="faTrash" />
+                        </button>
+
+                        <button type="button" class="btn icon-button-wrapper icon-button-2" (click)="triggerFileInput()">
+                            <fa-icon [icon]="faPen" />
+                        </button>
+                    </div>
                 }
                 @if (!croppedImage) {
-                    <div class="no-image"></div>
-                }
-                @if (!course.id) {
-                    <h4 id="jhi-course-heading-create" jhiTranslate="artemisApp.course.home.createLabel"></h4>
-                }
-                @if (course.id) {
-                    <h4 id="jhi-course-heading-edit" jhiTranslate="artemisApp.course.home.editLabel"></h4>
+                    <div [ngClass]="'artemisApp.course.imageCropper.uploadImageClass' | artemisTranslate" (click)="triggerFileInput()"></div>
                 }
+                <div class="row">
+                    <div class="col">
+                        @if (!course.id) {
+                            <div class="d-flex align-items-center justify-content-center">
+                                <h4 id="jhi-course-heading-create" jhiTranslate="artemisApp.course.home.createLabel"></h4>
+                            </div>
+                        }
+                        @if (course.id) {
+                            <div class="d-flex flex-column align-items-start">
+                                <h4 id="jhi-course-heading-edit" jhiTranslate="artemisApp.course.home.editLabel"></h4>
+                            </div>
+                        }
+                    </div>
```

Comment:
Can you explain why you're wrapping the labels in a flexbox?

Reply:
Removed